extension/src/popup/views/__tests__/Swap.test.tsx

```diff
@@ -51,7 +49,40 @@ export const swapMockBalances = {
       total: new BigNumber("333"),
       available: new BigNumber("333"),
     },
-  } as any as Balances,
```

Comment:
I removed the typecasting to simplify as it doesn't provide a ton of value in a test imo